llvm/test/LTO/X86/empty-triple.ll

```diff
@@ -0,0 +1,7 @@
+; RUN: llvm-as < %s >%t1
+; RUN: llvm-lto -o %t2 %t1  2>&1 | FileCheck --allow-empty %s
```

Comment:
This is in the X86 subdirectory, but I don't see where X86 use would come from. This is the odd case where "generic" target tests might be appropriate. Can you move this to the parent directory?